odk/odk.py

```diff
@@ -787,7 +787,7 @@ class Generator(object):
     """
 
     ## TODO: consider merging Generator and ExecutionContext?
-    context : ExecutionContext = ExecutionContext()
+    context : ExecutionContext = field(default_factory=ExecutionContext)
```

Comment:
Is this because of the newer dataclass version in the newer python?

Reply:
Yes.

The other fix that the `odk.py` script needed was the removal of a bogus escape sequence in one of the messages it displays to the user (`\@`).